Formula/tor.rb

```diff
@@ -39,15 +39,6 @@ def install
     system "make", "install"
   end
 
-  def caveats; <<-EOS.undent
```

Comment:
@DomT4 If someone is consistently misreading your communication there's at least a chance your communication could be improved. I've made a fairly simple request which you're free to ignore if you choose.